### PR TITLE
Update patch for taxjar credentials

### PR DESCRIPTION
**Pull request by lexerom**
## Summary
Add omitFromAddress to PatchTaxJarCredential. Add Conflict to POST and PATCH service credentials. Also add 'active' and 'inactive' statuses for patch bodies.

## Checklist

- [x] Writing style
- [x] API design standards

[Preview](https://preview.redoc.ly/all-rebilly/update-taxjar-credentials/tag/Service-credentials/operation/PatchServiceCredential)

### CODE REVIEW

**Review by noon-dawg — DISMISSED** [review]
Looks good on the docs side

**Review by ninjasimon — DISMISSED** [review]

**Review by i-yasko — DISMISSED** [review]

File: openapi/paths/service-credentials@{type}.yaml [hunk 1]
```diff
@@ -68,6 +68,8 @@ post:
       $ref: ../components/responses/Unauthorized.yaml
     '403':
       $ref: ../components/responses/Forbidden.yaml
+    '409':
+      $ref: ../components/responses/Conflict.yaml
```
Comment (lexerom) [inline openapi/paths/service-credentials@{type}.yaml:72]:
```suggestion
```

File: openapi/paths/service-credentials@{type}@{id}.yaml [hunk 2]
```diff
@@ -51,6 +51,8 @@ patch:
       $ref: ../components/responses/Forbidden.yaml
     '404':
       $ref: ../components/responses/NotFound.yaml
+    '409':
+      $ref: ../components/responses/Conflict.yaml
```
Comment (lexerom) [inline openapi/paths/service-credentials@{type}@{id}.yaml:55]:
```suggestion
```

**Comment by lexerom** [comment]
php please re-review

**Review by ashkarpetin — APPROVED** [review]

**Review by nanobash — APPROVED** [review]

**Pull request merged**